pkgs/by-name/op/ophcrack-cli/package.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, fetchpatch
+, autoreconfHook
+, libtool
+, pkg-config
+, zlib
+, openssl
+, freetype
+, gcc
+, expat
+}:
+
+stdenvNoCC.mkDerivation rec {
```

Comment:
why stdenvNoCC ?

Reply:
no particular reason. i just changed it to stdenv now.
for ophcrack gui #290496